kge/config-default.yaml

```diff
@@ -359,6 +359,21 @@ eval:
   # Other options
   pin_memory: False
 
+entity_ranking:
```

Comment:
Move this under eval for now. It's a good idea to have separate keys in general, but also other things would need to be moved (e.g., the filter splits). We can do this later in a separate PR.